tests/console/suseconnect_scc.pm

```diff
@@ -40,9 +40,11 @@ sub run {
     select_console('root-console');
     assert_script_run $cmd;
     assert_script_run 'SUSEConnect --list-extensions';
```

Comment:
As we do not have extensions, what does it yield over here?
Wondering if we should not add some validation of successful registration besides relying on L#41. Wdyt?

Reply:
assert_script_run will fail if it doesn't register properly.